src/main/java/org/opensearch/performanceanalyzer/reader/SearchBackPressureMetricsSnapShot.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.performanceanalyzer.reader;
+
+
+import java.sql.Connection;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jooq.BatchBindStep;
+import org.jooq.DSLContext;
+import org.jooq.Field;
+import org.jooq.Record;
+import org.jooq.Result;
+import org.jooq.SQLDialect;
+import org.jooq.impl.DSL;
+import org.opensearch.performanceanalyzer.commons.metrics.AllMetrics;
+
+public class SearchBackPressureMetricsSnapShot implements Removable {
+
+    // Logger for current class
+    private static final Logger LOG = LogManager.getLogger(SearchBackPressureMetricsSnapShot.class);
+
+    // entry point to interact with SQLite db
+    private final DSLContext create;
+
+    private final String tableName;
+    private List<Field<?>> columns;
+
+    // Create a table with specifed fields (columns)
+    public SearchBackPressureMetricsSnapShot(Connection conn, Long windowStartTime) {
```

Comment:
Add a Javadoc comment here describing the Table - columne name, sample data.

Reply:
Added new comments in latest commit.